package.json

```
@@ -49,16 +49,16 @@
     },
     "homepage": "https://github.com/ianpogi5/mocha-badge-generator",
     "dependencies": {
-        "badge-up": "^3.0.0",
+        "badge-up": "https://registry.npmjs.org/@rpl/badge-up/-/badge-up-3.0.0.tgz",
```

I think this should work:

```suggestion
        "badge-up": "npm:@rpl/badge-up@3.0.0",
```